### PR TITLE
fix accidental use of contents instead of baseMessage instance

### PR DESCRIPTION
**Pull request by KPostOffice**
Small fix to publish call && update CI files so that messaging mypy is used.

### CODE REVIEW

File: producer.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 from typing import Dict, Any, List
 
 import thoth.messaging.producer as producer
+from thoth.messaging import advise_justification_message
```
Comment (pacospace) [inline producer.py:29]: can you remove line below if not necessary?

**Review by pacospace — COMMENTED** [review]
Thank for this! Please make mypi happy :)

**Comment by KPostOffice** [comment]
@pacospace Leaving out config file param for another PR. Not sure why it was having trouble finding `mypy.ini`

**Review by fridex — APPROVED** [review]

**Pull request merged**